app/src/ui/diff/side-by-side-diff.tsx

```diff
@@ -337,6 +340,54 @@ export class SideBySideDiff extends React.Component<
     )
   }
 
+  private getRowWidth = (rows: readonly SimplifiedDiffRow[]): string => {
+    const largestBeforeLineNum = Math.max(
```

Comment:
I am not super familiar with the diff logic.. but I would suspect that the rows are ordered. If this is true, we could more efficiently just grab the last value form the rows away, correct?

@sergiou87 Maybe you are more familiar with this to weigh in.

Reply:
After poking through several repos and files it seems like the 2nd max call can be simplified as you said as the final value is always the last line in the diff. This makes me feel better because I didn't like the nested max statements, thank you!

Reply:
Sorry I completely missed this conversation 😱 

As @tidy-dev says, you shouldn't need to go through all rows. Just going to the last line of the last hunk (unless it's a "dummy" hunk just used for expansion in the UI) and going backwards looking for the first (last) instance of a new/old line, should be enough and much more efficient.